dictionaries/top-1000-words.json

```diff
@@ -930,7 +930,7 @@
 "A/PAOERS": "appears",
 "RO*EPL": "Rome",
 "HRAF": "laugh",
-"TPERGT": "forget",
+"TPORGT": "forget",
```

Comment:
What's your thinking for this change @paulfioravanti ? 

There's a set here: forget, forgets, forgot, forgotten, forgettable, unforgettable.

Some folks use FORGT for forgot and FERGT for forget.

I don't have a strong feeling on this, but I wanted to flag it.

Reply:
The original reason I proposed the change is that I read `TPERGT` as "fergt", with the "gt" sound together kind of filling in the missing "e" (kind of like with a regional US accent). For "forget", I felt that `TPORGT` spelt and sounded better, reading it in my head as "forgt".

To your point about "FORGT for forgot and FERGT for forget", I assume you mean:

- `TPERGT` for "forget"
- `TPORGT` for "forgot"

If so, then perhaps that's a mis-stroke entry to add(?), since Plover specifies the following outlines for "forgot":

```txt
forgot:
   TPAURGT
   TPO*RGT
   TPO*FRGT
   TPO*RBGT
   TPO*RLGT
```

As for the forget, forgets, forgot, forgotten, forgettable, unforgettable set, I get the following outlines from Plover:

```
forget:
   TPERGT
   TPORGT
   TPORLGT
forgets:
   TPORGTS
forgot:
   TPAURGT
   TPO*RGT
   TPO*FRGT
   TPO*RBGT
   TPO*RLGT
forgotten:
   TPO*RGD
   TPORGTD
   TPAURGTD
   TPO*RGTD
   TPA*URGTD
   TPORGT/-D
   TPAURGT/-D
   TPO*RGT/-D
   TPORGT/-PB
   TPORGT/*EPB
forgettable:
   TPERGT/-BL
   TPOR/TKPWET/-BL
unforgettable:
   UPB/TPORGT/-BL
   UPB/TPAUR/TKPWET/-BL
```

It would seem that there is some inconsistency of the use of `TPERGT` vs `TPORGT` within the set. Particularly strange to me would be the differences between "forgettable" and "unforgettable" and "forgotten"'s allowance of `TPORGTD` without an asterisk.

Maybe there's potentially some room to at least add some condensed strokes and then have the top-x dictionaries standardise on the preference of one scheme over another...? Any particular preferences?

Reply:
Sounds good, let's add some condensed strokes and then have top-x dictionaries standardise one the preference of one scheme.

I think the asterisk *might* be used here to indicate switching to past tense, where it's irrelevant in "forgotten" because "forgetted" or "forgetten" aren't words. Similar pattern in:

```
"TPAURG": "forgive",
"TPA*URG": "forgave",
"TPAURGD": "forgiven",
```

I can only guess the differences between "forgettable" and "unforgettable" are because other outlines like `"TPORGT/-BL": "forgettable",` would work without explicit entries using suffixes and orthographic rules. Not sure though.

Perhaps we run with:

```
"TPORGT": "forget",
"TPORGTS": "forgets",
"TPAURGT": "forgot",
"TPAURGTD": "forgotten",
"TPORGT/-BL": "forgettable", // new condensed entry using orthographic rules
"UPB/TPORGT/-BL": "unforgettable",
```

What do you think?

Reply:
Sounds good to me. I'll make those changes.

Reply:
Okay, changes made. There weren't as many top-x dictionary changes needed as I assumed, but at least there are now condensed stroke entries to bring balance to the forgettable/unforgettable inconsistencies 😄